osquery/tables/system/windows/processes.cpp

```diff
@@ -96,14 +119,41 @@ void genProcess(long pid, QueryData& results_data) {
       r["egid"] = "-1";
       r["sgid"] = "-1";
 
-      r["wired_size"] = "0";
-      r["resident_size"] = "0"; // Populate with WorkingSetSize (VT_BSTR)
-      r["phys_footprint"] = "0";
-
-      r["user_time"] = "0";
-      r["system_time"] = "0";
-      r["start_time"] = "0";
+      // We pre-populate the info, in the event the API calls fail.
+      r["user_time"] = "-1";
+      r["system_time"] = "-1";
+      r["start_time"] = "-1";
+      r["wired_size"] = "-1";
+      r["resident_size"] = "-1";
+      r["phys_footprint"] = "-1";
+
+      PROCESS_MEMORY_COUNTERS pmc;
+      auto ret = GetProcessMemoryInfo(hProcess, &pmc, sizeof(pmc));
+      if (ret != 0) {
+        r["wired_size"] = BIGINT(pmc.PagefileUsage);
+        r["resident_size"] = BIGINT(pmc.WorkingSetSize);
```

Comment:
I was basing the decision to poll the WorkingSetSize for RSS off of [this article](http://nadeausoftware.com/articles/2012/07/c_c_tip_how_get_process_resident_set_size_physical_memory_use), but that may or may not be right. I wasn't able to find much help from windows for building out that equivalence, but the numbers looked similar between running `osqueryi` in OS X vs running it in Windows.